### PR TITLE
Add support for WC subscriptions

### PR DESCRIPTION
**Pull request by mntzrr**
Related task: https://app.clickup.com/t/1p1z7j

### CODE REVIEW

**Review by NiklasHogefjord — APPROVED** [review]

File: classes/class-kp-subscriptions.php [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+			add_action( 'woocommerce_account_view-subscription_endpoint', array( $this, 'handle_redirect_from_change_payment_method' ) );
+
+			// Show the recurring token on the subscription page in the billing fields.
+			add_action( 'woocommerce_admin_order_data_after_billing_address', array( $this, 'show_recurring_token' ) );
+			// Ensure wp_safe_redirect do not redirect back to default dashboard or home page.
+			add_filter( 'allowed_redirect_hosts', array( $this, 'extend_allowed_domains_list' ) );
+
+		}
+
+		public function set_subscription_as_captured( $order_id ) {
```
Comment (MichaelBengtsson) [inline classes/class-kp-subscriptions.php:52]: Missing block comment for function

File: classes/requests/class-kp-requests-patch.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ *  The main class for PATCH requests.
+ */
+abstract class KP_Requests_Patch extends KP_Requests {
+
+	/**
+	 * Qliro_One_Request_Post constructor.
```
Comment (MichaelBengtsson) [inline classes/requests/class-kp-requests-patch.php:16]: Fix the comment here. Wrong class name

Comment (MichaelBengtsson) [reply classes/requests/class-kp-requests-patch.php:16]: Seems like we have the same in the post class as well, so can you fix that too while you are doing this one?

**Review by MichaelBengtsson — CHANGES_REQUESTED** [review]
Looks good except for the block comment on the request files.

**Pull request merged**